react-client/src/Components/App.js

```diff
@@ -5,16 +5,16 @@ import DomInfoDialogBox from "./DomInfoDialogBox";
 import { PRODUCTION_MODE } from "../keys";
 
 function App() {
-  // const [coordinates, setCoordinates] = useState({ top: 0, left: 0 });
+  const [notes, setNotes] = useState([]);
```

Comment:
what is notes for?

Reply:
setnotes  gnamit ko navariable sa test sample project  ko so i used it as  reference pra d ako mwala while implementing my work around